migration-engine/connectors/sql-migration-connector/src/sql_schema_calculator.rs

```diff
@@ -8,246 +8,300 @@ use datamodel::{
     Datamodel, DefaultValue, FieldArity, IndexDefinition, IndexType, ScalarType, ValueGenerator, ValueGeneratorFn,
```

Comment:
It's probably best to read this file out of the diff view, I completely reshuffled it.